mysql/content.md

```diff
@@ -1,49 +1,113 @@
+For more information and related downloads for MySQL Server and other MySQL products, please visit [www.mysql.com](http://www.mysql.com).
 
-Oracle Corporation and/or affiliates own the copyright and trademark for MySQL.
+# How to use this image
 
-> [wikipedia.org/wiki/MySQL](https://en.wikipedia.org/wiki/MySQL)
+## Start a `%%REPO%%` server instance
 
-%%LOGO%%
+Starting a MySQL instance is simple:
 
-# How to use this image
+docker run --name some-%%REPO%% -e MYSQL_ROOT_PASSWORD=my-secret-pw -d %%REPO%%:tag
+
+... where `some-%%REPO%%` is the name you want to assign to your container, `my-secret-pw` is the password to be set for the MySQL root user and `tag` is the tag specifying the MySQL version you want. See the list above for relevant tags.
+
+## Connect to MySQL from an application in another Docker container
+
+This image exposes the standard MySQL port (3306), so container linking makes the MySQL instance available to other application containers. Start your application container like this in order to link it to the MySQL container:
+
+docker run --name some-app --link some-%%REPO%%:mysql -d app-that-uses-mysql
+
+## Connect to MySQL from the MySQL command line client
+
+The following command starts another MySQL container instance and runs the `mysql` command line client against your original MySQL container, allowing you to execute SQL statements against your database instance:
+
+docker run -it --link some-%%REPO%%:mysql --rm %%REPO%% sh -c 'exec mysql -h"$MYSQL_PORT_3306_TCP_ADDR" -P"$MYSQL_PORT_3306_TCP_PORT" -uroot -p"$MYSQL_ENV_MYSQL_ROOT_PASSWORD"'
+
+... where `some-%%REPO%%` is the name of your original MySQL Server container.
+
+More information about the MySQL command line client can be found in the [MySQL documentation](http://dev.mysql.com/doc/en/mysql.html)
+
+## Container shell access and viewing MySQL logs
+
+The `docker exec` command allows you to run commands inside a Docker container. The following command line will give you a bash shell inside your `%%REPO%%` container:
+
+docker exec -it some-%%REPO%% bash
+
+The MySQL Server log is available through Docker's container log:
+
+docker logs some-%%REPO%%
 
-## start a `%%REPO%%` server instance
+## Using a custom MySQL configuration file
 
-	docker run --name some-%%REPO%% -e MYSQL_ROOT_PASSWORD=mysecretpassword -d %%REPO%%
+The MySQL startup configuration is specified in the file `/etc/mysql/my.cnf`. If you want to customize this configuration for your own purposes, you can create your alternative configuration file in a directory on the host machine and then mount this file in the appropriate location inside the `%%REPO%%` container, effectively replacing the standard configuration file.
```

Comment:
This should be talking about adding files in `/etc/mysql/conf.d/` instead, IMO, since then `my.cnf`'s rules don't have to be duplicated.  I don't know why the older version of this document didn't do that (I'd blame me :innocent:)...